### DISCUSSION TITLE
dConnection refusef

### DISCUSSION THREAD

**Discussion by kermit5242** [filamentphp/filament#11145] [Help]
### Package

Other

### Package Version

V3

### How can we help you?

Hi!
I have installed filament on a fresh Laravel install. Made migration, registered a user , Booted a web server. Every now and then, I get "ERR_CONNECTION_REFUSED" - the page doesn't load. The .env is configured with URL = 127.0.0.1, and I have also tried localhost. Sometimes it does work, but it's not consistent and I can't find a rule. Could it be cache, and if so how do I clear it? I have looked all over but I ca't find a solution. Thanks for any advice!

**Comment by leandrocfe** [comment]
Probably an issue in your local env. You can try demo because it uses sqlite
https://github.com/filamentphp/demo
